docs/HowTo/clusterconnectivity/confluent-cloud-kafka-cluster-ssl-protocol.md

```diff
@@ -57,7 +57,7 @@ Follow the steps below to configure and connect to a Confluent Cloud Kafka and K
    icon that is available on the right-hand side of each cluster
    row.
 9. Open the `application.properties` file located in the
-   [klaw/cluster-api/src/main/resources] directory.
+   [Klaw/cluster-api/src/main/resources] directory.
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?